README.md

```diff
@@ -1,2 +1,65 @@
-# pyronear-api
-API for wildfire prevention, detection &amp; monitoring
+# Pyronear API
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](LICENSE) [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3bea1a63e4aa44258cfd08831d713478)](https://www.codacy.com/gh/pyronear/pyronear-api/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=pyronear/pyronear-api&amp;utm_campaign=Badge_Grade)![Build Status](https://github.com/pyronear/pyronear-api/workflows/fastapi-project/badge.svg) [![codecov](https://codecov.io/gh/pyronear/pyronear-api/branch/master/graph/badge.svg)](https://codecov.io/gh/frgfm/Holocron) [![Docs](https://img.shields.io/badge/docs-available-blue.svg)](https://pyronear.github.io/pyronear-api)
+
+The building blocks of our wildfire detection & monitoring API.
+
+
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [License](#license)
+
+
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.6 (or more recent)
+- [pip](https://pip.pypa.io/en/stable/)
+
+### Installation
+
+You can clone and install the project dependencies as follows:
+
+```bash
+git clone https://github.com/pyronear/pyronear-api.git
+pip install -r pyronear-api/requirements.txt
```

Comment:
As far as requirements file is in src folder, should it be pip install -r pyronear-api/**_src_**/requirements.txt
```suggestion
 pip install -r pyronear-api/src/requirements.txt
```

Reply:
Nice catch, just fixed it!